src/wp-admin/user-new.php

```diff
@@ -443,7 +443,7 @@
 <table class="form-table" role="presentation">
 	<tr class="form-field form-required">
 		<th scope="row"><label for="adduser-email"><?php echo $label; ?></label></th>
```

Comment:
Hey @monzuralam, your PR looks good to me. 
Also, I think U can use `esc_html` for the `$label` just one line above ( line 445 )

Reply:
Hey @hz-tyfoon, Thanks bhaia.
The label escaping issue has been fixed.